_includes/specials/policy/en/06-consistency.md

```diff
@@ -0,0 +1,101 @@
+only accepted transactions with that `nVersion`.  As most peer-to-peer
+relationships would have incompatible policies, transactions would not
+propagate to miners.
+
+On the other end of the spectrum, identical policies across the
+network help converge mempool contents. A network with matching
+mempools relays transactions the most smoothly, and is also ideal for
+[fee estimation][policy04] and [compact block relay][policy01] as
+mentioned in previous posts.
+
+Given the complexity of mempool validation and the difficulties that
+arise from policy disparities, Bitcoin Core has [historically been
+conservative][aj mempool consistency] with the configurability of
+policies. While users are able to easily tweak the way sigops are
+counted (`bytespersigop`) and limit the amount of data embedded
+in`OP\_RETURN` outputs (`datacarriersize` and `datacarrier`), they
+cannot opt out of the 400,000Wu maximum standard weight or apply a
+different set of fee-related RBF rules without changing the source
+code.
+
+Some of Bitcoin Core’s policy configuration options exist to
+accommodate the difference in node operating environments and purposes
+for running a node. For example, a miner’s hardware resources and
+purpose for keeping a mempool differ from a day-to-day user running a
+lightweight node on their laptop or Raspberry Pi. A miner may opt to
+increase their mempool capacity (`maxmempool`) or expiration timeline
+(`mempoolexpiry`) to store low feerate transactions during peak
+traffic, and then mine them later when traffic dies down. Websites
+providing visualizations, archives, and network statistics may run
+multiple nodes to collect as much data as possible and also display
+default mempool behavior.
+
+On an individual node, the choice of mempool capacity affects the
+availability of fee-bumping tools.  When the mempool minimum feerates
+rise due to transaction submissions exceeding the default mempool
+size, transactions purged from the “bottom” of the mempool and new
+ones that are below this feerate can no longer be fee-bumped using
+CPFP.
+
+On the other hand, since the inputs used by the purged transactions
+are no longer spent by any transactions in the mempool, it may be
+possible to fee-bump via RBF when it wasn’t before. The new
+transaction isn’t actually replacing anything in the node’s mempool,
+so it doesn’t need to consider the usual RBF rules. However, nodes
+that haven’t evicted the original transaction (because they have a
+larger mempool capacity) treat the new transaction as a proposed
+replacement and require it to abide by the RBF rules. If the purged
+transaction was not signaling BIP125 replaceability, or the new
+transaction’s fee does not meet RBF requirements despite being high
+feerate, the miner may not accept their new transaction. Wallets must
+handle purged transactions carefully: the transaction’s outputs cannot
+be considered available for spending, but the inputs are similarly
+unavailable to reuse.
```

Comment:
I'm not quite following this, is it because the miner may not have purged those transactions (due to miners possibly running a larger mempool)? Also, to check my understanding, "the inputs are ..." -- really you mean the outputs that those inputs spend? The actual inputs have been purged, IIUC.